MenuAPI/MenuController.cs

```diff
@@ -1112,6 +1112,8 @@ internal static async Task DrawInstructionalButtons()
                 await Delay(0);
             }
 
+            DrawScaleformMovieFullscreen(_scale, 255, 255, 255, 0, 0);
```

Comment:
Not entirely sure why this is added here, shouldn't line 1152 be changed instead?

Reply:
The _first_ draw sets up the display config which is used by the functions called below, as seen in the vMenu PR this 0-alpha draw is a 'dummy' so that it'll set up display config.